app/controllers/home_controller.rb

```diff
@@ -1,4 +1,35 @@
-class HomeController < ApplicationController
-  def index
-  end
-end
+include Rails.application.routes.url_helpers
+
+class HomeController < ApplicationController
+  @@nav_links = {

```

Comment:
I like what you did with the loop, but is it a problem the nav_links only appears to be in the HomeController?  Can the donate and apply pages access this variable?  They need nav bars too.

Reply:
done

Reply:
Why is this done?  Did you confirm it can be accessed by the other controllers?  I don't see that you added it to the other controllers.  Also, it looks like the donate page is still missing a nav bar.

Reply:
actually done now. It's in ApplicationController (which other controllers inherit from)